sbi/inference/abc/mcabc.py

```diff
@@ -158,13 +166,24 @@ def __call__(
         else:
             theta_adjusted = theta_accepted
```

Comment:
Maybe rename `theta_adjusted` to `final_theta` (similar to how `final_particles` is used for SMC-ABC)?